public/js/chessClasses/gameManager.js

```diff
@@ -33,4 +51,33 @@ class GameManager {
     this.board.setSquare( endCoord, this.board.getSquare( startCoord ))
     this.board.setSquare( startCoord, false )
   }
+
+  placeScattered () {
+    this.board.setSquare( 'e7', new Piece( KNIGHT, BLACK ) )
```

Comment:
The [magic string](https://en.wikipedia.org/wiki/Magic_string) jumps out at me as an opportunity for bugs - a typo could introduce subtle misbehaviors that would need to be tracked back to a string.  Consider replacing this with constants defined on the board class, or some other means of generically telling the board to do its job.